docs/document/docs/components/desktop/date-picker.md

```diff
@@ -0,0 +1,74 @@
+---
+id: date-picker
+title: DatePicker
+sidebar_label: DatePicker
+---
+
+## Overview
+
+The DatePicker component allows the user to display a input area and date selection calendar.
+
+<iframe src="https://kuc-storybook.netlify.app/iframe.html?id=desktop-date-picker--document" title="date picker image" height="80px"></iframe>
+
+---
+
+## Specification
+
+### Property
+
+Here is a list of properties that can be used for modifying the component:
+
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| className | string | ""  | Component class name | |
+| error | string | ""  | Text to be displayed in error | Error will not be displayed if unspecified or left empty |
+| id | string | ""  | Component id name | |
+| label | string | ""  | Label for the component | Label will not be displayed if unspecified or left empty |
+| language | string | "auto"  | Language setting | Available options: "auto", "en", "ja", "zh"<br>If setting "auto", it will be according to the HTML lang setting (If the lang setting is other than "en"/"zh"/"ja", the language setting will be "en") |
+| value | string | ""  | Text to be displayed | Format is YYYY-MM-DD<br>The below date can be used (it will be converted to YYYY-MM-DD internally): 2021/2021-1/2021-01/2021-1-9/21-01-01<br>If the year is less than 4 characters, 0 will be added to the beginning<br>If the date and/or month is ignored, it will be supplemented with 01<br>If the date and/or month has only 1 character, 0 will be added to the beginning<br>Will result an error if setting invalid format or value |
```

Comment:
<li>2021</li><li>2021-1</li><li>2021-01</li><li>2021-1-9</li><li>21-01-01</li>
i think you can put <li> tag, it is easy to read
<img width="437" alt="Screen Shot 2022-01-05 at 10 35 22" src="https://user-images.githubusercontent.com/12889051/148156808-3e3107a3-a488-4e1c-bdbf-6e36a71813b3.png">